demos/gallery/samples/common/CanvasWidget.js

```diff
@@ -0,0 +1,34 @@
+import { CanvasWidget } from "@hpcc-js/common";
+
+const data = [
+    ["A", 100],
+    ["B", 162],
+    ["C", 262],
+];
+class CanvasExample extends CanvasWidget{
+    constructor(){
+        super();
+    }
+    enter(domNode, element){
+        super.enter.apply(this, arguments);
```

Comment:
change to `super.enter(domNode, element)` and do the same of all siimlar examples